packages/core/src/routes/interaction/utils/single-sign-on-guard.ts

```diff
@@ -15,10 +16,14 @@ import assertThat from '#src/utils/assert-that.js';
 // Guard the SSO only email identifier
 export const verifySsoOnlyEmailIdentifier = async (
   { getAvailableSsoConnectors }: SsoConnectorLibrary,
-  identifier: IdentifierPayload
+  identifier: IdentifierPayload | SocialUserInfo
 ) => {
```

Comment:
I have a question: How do we define if the social email is SSO `only`?
It seems we'll consider it as `SSO only` if the SSO domain contains the email returned by social, isn't it?

Reply:
Yes